payment-instruments/trustly/other-features.md

```diff
@@ -104,6 +106,8 @@ Content-Type: application/json
 |                  | └─➔&nbsp;`productCategory`   | `string`      | A product category or number sent in from the payee/merchant. This is not validated by Swedbank Pay, but will be passed through the payment process and may be used in the settlement process.                                                                                                     |
 |                  | └─➔&nbsp;`orderReference`    | `String(50)`  | The order reference should reflect the order reference found in the merchant's systems.                                                                                                                                                                                                            |
 |                  | └─➔&nbsp;`subsite`           | `String(40)`  | {% include field-description-subsite.md %}                                                                                                                                        |
+| {% icon check %} | └➔&nbsp;`payer`               | `string`     | The `payer` object, containing information about the payer.                                                                                                                                                                                                                                          |
+| {% icon check %} | └─➔&nbsp;`payerReference`                | `string`     | {% include field-description-payer-reference.md documentation_section="trustly" %}                                                                                                                                                                                                                                                           |
```

Comment:
Hm, I don't think `payer` or `payerReference` are required? Please remove `{% icon check %}` everywhere if they are indeed optional.

Reply:
The internal documentation states that this is optional, but mandatory when using paymentToken and createPaymentToken, so I guess there is quite a bit of case-by-case checkups here.

Reply:
Scratch that. Changed the include and removed the icons everywhere.